sdk/src/asset_handlers/mp3_io.rs

```diff
@@ -344,6 +344,18 @@ impl AssetIO for Mp3IO {
     fn supported_types(&self) -> &[&str] {
         &SUPPORTED_TYPES
     }
+
+    fn supports_stream(&self, stream: &mut dyn CAIRead) -> Result<bool> {
+        stream.rewind()?;
+
+        let mut header = [0u8; 3];
+        stream.read_exact(&mut header)?;
+
+        Ok(
+            (header[0] == 0x49 && header[1] == 0x44 && header[2] == 0x33)
```

Comment:
If we think more checks are needed, I would not add it here now. Either split it out, or do the fill check. I'd rather we be less restrictive to possibly avoid breaking edge cases now - or due the compelte due diligence.

@ok-nick What makes you say more checks may be needed for this case?

Reply:
Ok!